Tests/FiftyOne.Pipeline.Engines.Tests/Services/DataUpdateServiceTests.cs

```diff
@@ -93,6 +96,33 @@ public void Init()
                 _fileSystem.Object,
                 _timerFactory.Object);
 
+            OnDataUpdateServiceCreated();
+        }
+
+        private void OnDataUpdateServiceCreated()
+        {
+            _dataUpdate.DebugLoggingEnabled = true;
+
+            Func<string, Action> lockLoggerFor = reason => () =>
+            {
+                _logger.LogDebug($"Locking logger for {reason}");
+                Monitor.Enter(_logger);
+                _logger.LogDebug($"Locked logger for {reason}");
+            };
+            Func<string, Action> unlockLoggerFrom = reason => () =>
+            {
+                _logger.LogDebug($"Unlocking logger from {reason}");
+                Monitor.Exit(_logger);
+                _logger.LogDebug($"Unlocked logger from {reason}");
+            };
+
+            _dataUpdate.OnTimeredCheckForUpdateEntered += lockLoggerFor("timered CheckForUpdate");
```

Comment:
cosmetics but still, maybe it is worth creating a string constant "timered CheckForUpdate" - for it is repeated lower incode.  and the same thing for "DatafileUpdated" - I assume the lock/unlock should be done with the same parameter?